### PR TITLE
archive generated website

### PR DESCRIPTION
**Pull request by sandrobonazzola**
Changes proposed in this pull request:

- archive generated website allowing to review the results.
- document how to consume archived artifacts

I confirm that this pull request was submitted according to the [contribution guidelines](https://github.com/oVirt/ovirt-site/blob/master/CONTRIBUTING.md): @sandrobonazzola 

This pull request needs review by: @duck-rh @stoobie

### CODE REVIEW

**Comment by stoobie** [comment]
@sandrobonazzola Is this correct: 

This PR modifies website_tests.yml such that any time the website is generated, a .zip file containing the website is generated.

Can you explain, as a result of this changes, what a writer needs to do in order to view a preview of changes made in a PR?

**Comment by duck-rh** [comment]
> @sandrobonazzola Is this correct:
> 
> This PR modifies website_tests.yml such that any time the website is generated, a .zip file containing the website is generated.
> 
> Can you explain, as a result of this change, what a writer needs to do in order to view a preview of changes made in a PR?

I had the same question, but the reply lies into the action's documentation:
  https://github.com/actions/upload-artifact#where-does-the-upload-go

**Review by stoobie — CHANGES_REQUESTED** [review]
I downloaded the .zip file and opened it. I can view the HTML, but it has no styling attached to it. This doesn't work as a solution for previewing doc changes.

**Comment by sandrobonazzola** [comment]
In order to get the styking you need to serve the site: "python3 -m http.server" is enough to serve it

**Review by duck-rh — CHANGES_REQUESTED** [review]
It's a very good idea.

The question left by @stoobie is legitimate which leads me to the request to add documentation for contributors in `README.md`.

There is also a caveat: various redirections are done on Apache side in `../ovirt-site/source/.htaccess` so this does not fully represent the result once in production.

**Comment by sandrobonazzola** [comment]
ok, let me work on the readme.

**Comment by sandrobonazzola** [comment]
@duck-rh , @stoobie I updated the README.md, please review :-)

**Review by duck-rh — APPROVED** [review]
LGTM

File: README.md [hunk 1]
```diff
@@ -11,19 +11,32 @@ To submit a bug or suggest a documentation fix/enhancement for a specific page,
...
-You can also clone and fork the repository like any other GitHub project, if you are more comfortable with that. See [CONTRIBUTING.md](CONTRIBUTING.md) for instructions on building and testing the site locally.
+You can also clone and fork the repository like any other GitHub project, if you are more comfortable with that.
+See [CONTRIBUTING.md](CONTRIBUTING.md) for instructions on building and testing the site locally.
 
 See [MAINTAINING.md](MAINTAINING.md) for information about site maintainers.
 
 See [README_ORGANIZATION.md](README_ORGANIZATION.md) for information about how the source code is organized.
 
-All content in the website is written in Markdown. For information on formatting Markdown, see [Markdown Basics](https://help.github.com/articles/getting-started-with-writing-and-formatting-on-github/).
+Most of the content in the website is written in Markdown.
```
Comment (stoobie) [inline README.md:21]: Product documentation is written in Asciidoc.

File: README.md [hunk 2]
```diff
@@ -11,19 +11,32 @@ To submit a bug or suggest a documentation fix/enhancement for a specific page,
...
+For information on formatting Markdown, see [Markdown Basics](https://help.github.com/articles/getting-started-with-writing-and-formatting-on-github/).
```
Comment (stoobie) [inline README.md:22]: For information on authoring in Asciidoc, see https://asciidoctor.org/docs/asciidoc-writers-guide/

File: README.md [hunk 3]
```diff
@@ -11,19 +11,32 @@ To submit a bug or suggest a documentation fix/enhancement for a specific page,
...
 
 - From the GitHub repo, navigate to the relevant directory and click the New File button at the top-right corner of the pane.
 
+After you submit a PR to the ovirt-site reposiory, automation will verify your submission. If no error is found you'll see a green symbol and a text saying
+"All checks have passed". You can inspect the generated website by downloading the site-without-markdown.zip file from the automation system.
+In order to do so, click on the "Checks" tab within your PR and then click on "Website Tests" link on the left bar.
+You can find "site-without-markdown" artifact at the bottom of the page.
+
+Once downladed the zip file, in order to check how the website looks like you need to extract the archive and serve the extracted directory with a web server.
+An easy way to do it is using python: `python3 -m http.server` from the web site directory. This won't provide a full replica of what will happen on the real website
```
Comment (stoobie) [inline README.md:36]: I was able to run the server and load the site home page, but I couldn't open any documentation, and there was no styling on the site. When I clicked on Documentation, I got:
image

**Pull request merged**